sub-packages/bionemo-evo2/tests/bionemo/evo2/test_evo2.py

```diff
@@ -175,19 +174,6 @@ def test_golden_values_top_k_logits_and_cosine_similarity_7b(seq_len: int = 8_19
         attention_mask = None
         outputs = model(input_ids=input_ids, position_ids=position_ids, attention_mask=attention_mask)
         gold_standard_no_fp8_tensor = torch.load(gold_standard_no_fp8).to(device=outputs.device, dtype=outputs.dtype)
-        is_fp8_supported, compute_capability, device_info = check_fp8_support(device.index)
```

Comment:
is there any case where the strict test is satisfied, ie A100 on x86? maybe instead of removing this check entirely, we should scope the inf statement or add some tolerance?

Reply:
cc @jstjohn

Reply:
I'm ok, between this and the AUC test we added into the notebook we're probably ok.